main.rb

```diff
@@ -1,10 +1,22 @@
 require './person'
 require './capitalize_decorator'
 require './trimmer_decorator'
+require './student'
+require './classroom'
 
-person = Person.new(22, 'maximilianus')
-p person.correct_name
-capitalized_person = CapitalizeDecorator.new(person)
-p capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-p capitalized_trimmed_person.correct_name
+# person = Person.new(22, 'maximilianus')
+# p person.correct_name
+# capitalized_person = CapitalizeDecorator.new(person)
+# p capitalized_person.correct_name
+# capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+# p capitalized_trimmed_person.correct_name
```

Comment:
- [ ] [**OPTIONAL**] Kindly, remove the commented code from your codebase to make your code clean.